moveit_core/kinematic_constraints/src/kinematic_constraint.cpp

```diff
@@ -58,6 +58,58 @@ static double normalizeAngle(double angle)
   return v;
 }
 
+// Normalizes an angle to the interval [-pi, +pi] and then take the absolute value
+// The returned values will be in the following range [0, +pi]
```

Comment:
Minor correction:

// Normalizes the absolute value of an angle to [0, +pi]
// Returned values will be in the range [0, +pi]